File: Java/FizzBuzz.java
package Java;

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    System.out.print("How many numbers to display: ");
    String numcount = input.nextLine();
    for (int i = 1; i <= Integer.parseInt(numcount); i++) {
        if (i % 3 == 0 && i % 5 == 0) {
            System.out.println("FizzBuzz");
        } else if (i % 3 == 0) {
            System.out.println("Fizz");
        } else if (i % 5 == 0) {
            System.out.println("Buzz");
        } else {
            System.out.println(i);
        }
    }
    input.close();
    }
}

File: Go/fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var numcount string
	fmt.Print("How many numbers to display: ")
	fmt.Scanf("%s", &numcount)
	intnumcount, err := strconv.Atoi(numcount)
	if err != nil {
		fmt.Println("Error: ", err)
		return
	}
	for i := 1; i <= intnumcount; i++ {
		if i%3 == 0 && i%5 == 0 {
			fmt.Println("FizzBuzz")
		} else if i%3 == 0 {
			fmt.Println("Fizz")
		} else if i%5 == 0 {
			fmt.Println("Buzz")
		} else {
			fmt.Println(i)
		}
	}
}
